src/dumpMetadata/models/dumpMetadataManager.ts

```diff
@@ -40,7 +40,10 @@ export class DumpMetadataManager {
   }
 
   private getUrlHeader(): string {
-    const { protocol, host, port } = this.objectStorageConfig;
+    const { protocol, host, projectId, port } = this.objectStorageConfig;
+    if (projectId != undefined) {
+      return `${protocol}://${host}/${projectId}:${port}`;
```

Comment:
what if `projectId` is empty, we would still have a semicolon, I do not think it is the desired behavior

Reply:
I've added tests for the case `projectId` is not defined as well and changed the default value in `default.json` to be undefined.
in the case of an empty string the `projectId` will be accounted as undefined.